build-novm.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+__usage() { echo "Usage: ${0}:"; echo "  ${0} <all|project [project] ...>"; exit 1; };
+
+if [[ ${COOPR_BUILD_PROJECTS} ]]; then
+  __OUTPUT="Building ${COOPR_BUILD_PROJECTS}"
+elif [[ ${#} -eq 1 ]] && [[ ${1} == 'all' ]]; then
+  COOPR_BUILD_PROJECTS="coopr-cli coopr-provisioner coopr-server coopr-ui"
+  __OUTPUT="Building all (${COOPR_BUILD_PROJECTS})"
+elif [[ ${#} -ge 1 ]]; then
```

Comment:
I would recommend "elif [[ ${#} -gt 1 ]]; then" here, even though it will work the other way.